src/app/components/client/ExposureCard.tsx

```diff
@@ -132,48 +122,66 @@ export const ExposureCard = (props: Props) => {
             detailsOpen ? styles.isOpen : ""
           }`}
         >
-          <p>
-            This site is selling and publishing{" "}
-            <a href={exposureDetailsLink}>
-              details about you.{" "}
-              <span>
-                <OpenInNew alt="" width="15" height="14" />
-              </span>
-            </a>{" "}
-            Remove this profile to protect your privacy.
-          </p>
+          {isScanResult(props.exposureType) ? 
+          // Data broker content
+          <div>
+            <p>
+              {l10n.getFragment("exposure-card-description-info-for-sale-part-one", 
+              {
+                elems: {"data_broker_link": <a href={exposureDetailsLink}/>}
+              })}
+              <a href={exposureDetailsLink}>
+                <span>
+                  <OpenInNew alt={l10n.getString("open-in-new-tab-alt")} width="13" height="13" aria-label={l10n.getString("open-in-new-tab-aria-label")} />
+                </span>
+              </a>
+              {" "}{l10n.getString("exposure-card-description-info-for-sale-part-two")}
+            </p>
+          </div> : 
+          // Data breach content
+          <div>
+            <p>
+              {l10n.getFragment("exposure-card-description-data-breach-part-one", 
+              {
+                vars: {
+                  data_breach_company: exposureName,
+                  data_breach_date: props.exposureType.BreachDate
+                },
+                elems: {"data_breach_link": <a href={exposureDetailsLink}/>}
+              })}
+              <a href={exposureDetailsLink}>
+                <span>
```

Comment:
Same Q here re: possibly unneeded `<span>`.

And not sure which PR it was in, but I believe @flodolo shared some concerns about these strings basically being 2-3 strings glued together, which doesn't often play well w/ localizering. (applies to this block/div and others above where we seem to be splitting into `*-part-one` and `*-part-two`).

Reply:
It really doesn't, quite the opposite. You're enforcing the English structure of the sentence on all locales, and hard-coding characters to do it (a colon here, a space after).